cmd/protoc-gen-gorums/dev/server.go

```diff
@@ -61,33 +47,14 @@ func (s *orderingServer) NodeStream(srv ordering.Gorums_NodeStreamServer) error
 		}
 	}()
 
-	go func() {
-		for {
-			select {
-			case <-ctx.Done():
-				return
-			case req := <-ordered:
-				handleMsg(req.msg, req.info)
-			}
-		}
-	}()
-
 	for {
 		req := newGorumsMessage(false)
```

Comment:
Shouldn't this be `true`?

Reply:
The `reply` field in the `gorumsMessage` type is used to tell the `gorumsUnmarshal` function whether it should create a new instance of the request type or the response type. On the server, we want to unmarshal the Request type, and hence it is set to `false`. I suppose I could create some constants to use in place of `true/false`. Might make it easier to read.

Reply:
Or perhaps it could be an argument to the `gorumsUnmarshal` function instead...

Reply:
It was just confusing when `reply=false` was used to mark a response message. You could  also just change the name of the input to `request` instead of `reply`. Consts could also work.  Didn’t look at the last suggestion in depth. So maybe that’s better...

Reply:
I have added the consts `gorumsRequest` and `gorumsResponse`, which are used instead of a boolean.